packages/client/README.md

```diff
@@ -236,7 +236,7 @@ const sub2 = await streamr.resend(
             timestamp: (Date.now() - 1000 * 60 * 5), // 5 minutes ago
             sequenceNumber: 0, // optional
         },
-        publisher: '0x12345...', // optional
+        publisher: '0x1234567890123456789012345678901234567890', // optional
```

Comment:
I'd use the shorter form, it is more readable to me. Or am I contradictory to some of my previous commit?